config/data/holes.toml

```diff
@@ -3338,6 +3338,20 @@ preamble = '''
 <p>Print all the sphenic numbers from <b>1</b> to <b>10,000</b> inclusive, each on their own line.
 '''
 
+['Star Wars GPT']
+experiment = 0
```

Comment:
This ought to have a real issue created so people can discuss the hole when it's live but experimental.